cmd/athenz-sia/main.go

```diff
@@ -15,35 +15,34 @@
 package main
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"math/rand"
 	"os"
 	"os/signal"
 	"path/filepath"
 	"syscall"
+	"time"
 
+	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/certificate"
 	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/config"
-	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/identity"
+	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/healthcheck"
+	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/metrics"
+	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/token"
 	"github.com/AthenZ/k8s-athenz-sia/v3/pkg/version"
 	"github.com/AthenZ/k8s-athenz-sia/v3/third_party/log"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
 const serviceName = "athenz-sia"
 
-var (
-	APP_NAME   = version.APP_NAME
-	VERSION    = version.VERSION
-	BUILD_DATE = version.BUILD_DATE
-)
-
 // printVersion returns the version and the built date of the executable itself
 func printVersion() {
-	if VERSION == "" || BUILD_DATE == "" {
+	if version.VERSION == "" || version.BUILD_DATE == "" {
```

Comment:
@WindzCUHK Maybe this is a little relevant from this PR's context and should have been pointed in https://github.com/AthenZ/k8s-athenz-sia/pull/105/files but, but the building environment parameter is not only for version management and the perspective should not be narrowed for only version management, thus I suppose the struct itself should be treated as a more generic usage.(e.g. arguments.go)

Reply:
refactor in different PR?

- `version.go` (appName, version, etc.): https://github.com/AthenZ/k8s-athenz-sia/pull/105
- `cache.go` (pod, ns, etc.): https://github.com/AthenZ/k8s-athenz-sia/pull/103

Reply:
fix in PR: https://github.com/AthenZ/k8s-athenz-sia/pull/108/commits/a04d5afd0a834717b01aec3bd94dbb90f9c2432c